README.md

```diff
@@ -3,17 +3,29 @@ Demo of media pipe python integration with Django
 
 ## Server Setup
 
-### GCP 
-- Temporary Server (Dwayne's Personal) `gcloud beta compute ssh --zone "us-east4-c" "mediapipe-server" --project "mystical-slate-241320"`
+### Docker / Local
+
+#### Mediapipe Container
+- Taken from https://google.github.io/mediapipe/getting_started/install.html
+- In a sibling folder, `git clone git@github.com:mitmedialab/mediapipe.git`
+- `docker build --tag=mediapipe .`
+- Quick Test (optiona1)
+  - `docker run -it --name mediapipe mediapipe:latest`
+  - `root@bca08b91ff63:/mediapipe# GLOG_logtostderr=1 bazel run --define MEDIAPIPE_DISABLE_GPU=1 mediapipe/examples/desktop/hello_world:hello_world`
```

Comment:
I slacked you but unfortunately having trouble running these commands :( So sorry!